bin/commands/init/security/index.sh

```diff
@@ -11,8 +11,190 @@
+  fi
+  _CEE_RUNOPTS="XPLINK(ON),HEAPPOOLS(OFF),HEAPPOOLS64(OFF)" ${ZWE_zowe_runtimeDirectory}/bin/utils/configmgr -script "${ZWE_zowe_runtimeDirectory}/bin/commands/init/security/cli.js"
+else
+
+###############################
+# Old 3.2 code follows
+
+
+
+
+print_level1_message "Run Zowe security configurations"
+
+###############################
+# constants
+
+###############################
+# validation
+require_zowe_yaml "skipnode"
+
+# read prefix and validate
+prefix=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.prefix")
+if [ -z "${prefix}" ]; then
+  print_error_and_exit "Error ZWEL0157E: Zowe dataset prefix (zowe.setup.dataset.prefix) is not defined in Zowe YAML configuration file." "" 157
+fi
+# read JCL library and validate
+jcllib=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.dataset.jcllib")
+if [ -z "${jcllib}" ]; then
+  print_error_and_exit "Error ZWEL0157E: Zowe custom JCL library (zowe.setup.dataset.jcllib) is not defined in Zowe YAML configuration file." "" 157
+fi
+security_product=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.product")
+if [ -z "${security_product}" ]; then
+  security_product=RACF
+fi
+security_groups_admin=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.groups.admin")
+if [ -z "${security_groups_admin}" ]; then
+  security_groups_admin=${ZWE_PRIVATE_DEFAULT_ADMIN_GROUP}
+fi
+security_groups_stc=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.groups.stc")
+if [ -z "${security_groups_stc}" ]; then
+  security_groups_stc=${ZWE_PRIVATE_DEFAULT_ADMIN_GROUP}
+fi
+security_groups_sysProg=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.groups.sysProg")
+if [ -z "${security_groups_sysProg}" ]; then
+  security_groups_sysProg=${ZWE_PRIVATE_DEFAULT_ADMIN_GROUP}
+fi
+security_users_zowe=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.users.zowe")
+if [ -z "${security_users_zowe}" ]; then
+  security_users_zowe=${ZWE_PRIVATE_DEFAULT_ZOWE_USER}
+fi
+security_users_zis=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.users.zis")
+if [ -z "${security_users_zis}" ]; then
+  security_users_zis=${ZWE_PRIVATE_DEFAULT_ZIS_USER}
+fi
+security_stcs_zowe=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.stcs.zowe")
+if [ -z "${security_stcs_zowe}" ]; then
+  security_stcs_zowe=${ZWE_PRIVATE_DEFAULT_ZOWE_STC}
+fi
+security_stcs_zis=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.stcs.zis")
+if [ -z "${security_stcs_zis}" ]; then
+  security_stcs_zis=${ZWE_PRIVATE_DEFAULT_ZIS_STC}
+fi
+security_stcs_aux=$(read_yaml_configmgr "${ZWE_CLI_PARAMETER_CONFIG}" ".zowe.setup.security.stcs.aux")
+if [ -z "${security_stcs_aux}" ]; then
+  security_stcs_aux=${ZWE_PRIVATE_DEFAULT_AUX_STC}
+fi
+
+###############################
+# prepare ZWESECUR JCL
+print_message "Modify ZWESECUR"
```

Comment:
We have to alter the job header and remove "new" variable:
`//ZWESECUR JOB {zowe.setup.jcl.header}` -> `//ZWESECUR JOB`